docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -156,18 +155,19 @@ Search connectors are only needed when importing external data into the Knowledg
 
 {enterprise-search-ref}/connectors.html[Connectors] allow you to index content from external sources thereby making it available for the AI Assistant. This can greatly improve the relevance of the AI Assistant’s responses. Data can be integrated from sources such as GitHub, Confluence, Google Drive, Jira, AWS S3, Microsoft Teams, Slack, and more.
 
-These connectors are managed under *Search* -> *Content* -> *Connectors* in {kib}, they are outside of the {observability} Solution, and they require an {enterprise-search-ref}/server.html[Enterprise Search] server connected to the Elastic Stack.
+These connectors are managed under the Search Solution in {kib}, they are outside of the {observability} Solution, and they require an {enterprise-search-ref}/server.html[Enterprise Search] server connected to the Elastic Stack.
```

Comment:
```suggestion
These connectors are managed under the Search Solution in {kib}, and they require an {enterprise-search-ref}/server.html[Enterprise Search] server connected to the Elastic Stack.
```

Do you think the `they are outside of the {observability} Solution` could be omitted? (It was added by me recently but I believe specifying it doesn't really add any value). Whatever you prefer.

Reply:
I agree...we can leave it out.